src/libraries/System.Runtime.Intrinsics/ref/System.Runtime.Intrinsics.cs

```diff
@@ -4651,6 +4651,15 @@ public new abstract partial class Arm64 : System.Runtime.Intrinsics.Arm.AdvSimd.
         public static System.Numerics.Vector<uint> CreateBreakBeforePropagateMask(System.Numerics.Vector<uint> mask, System.Numerics.Vector<uint> left, System.Numerics.Vector<uint> right) { throw null; }
         public static System.Numerics.Vector<ulong> CreateBreakBeforePropagateMask(System.Numerics.Vector<ulong> mask, System.Numerics.Vector<ulong> left, System.Numerics.Vector<ulong> right) { throw null; }
 
+        public static unsafe System.Numerics.Vector<sbyte> CreateBreakPropagateMask(System.Numerics.Vector<sbyte> totalMask, System.Numerics.Vector<sbyte> fromMask) { throw null; }
```

Comment:
nit: The `/ref/*.cs` files in particular are sometimes handled (or regenerated) by tools and they expect these to be ordered alphabetically and with no newlines between the groups

For the alphabetical order they consider the .NET type names and not the C# keyword names, so the order is:
* `Byte` (byte)
* `Char` (char)
* `Decimal` (decimal)
* `Double` (double)
* `Int16` (short)
* `Int32` (int)
* `Int64` (long)
* `IntPtr` (nint)
* `SByte` (sbyte)
* `Single` (float)
* `UInt16` (ushort)
* `UInt32` (uint)
* `UInt64` (ulong)
* `UIntPtr` (nuint)

Reply:
We will make follow-up work to clean it up.
https://github.com/dotnet/runtime/issues/104833
https://github.com/dotnet/runtime/issues/104834